SDK 2.3.0  default RAM memory consumption increase

After updating to the Pico SDK 2.3.0, I am unable to build my project due to over ~40KB increase in RAM consumption. Does anyone know where I can find documentation on these changes, or the best way to identify what is causing this increase?
```
 cd /home/shtirlic/projects/8086/picocalc_x86/build && /home/shtirlic/.local/bin/picotool uf2 convert --quiet /home/shtirlic/projects/8086/picocalc_x86/build/picocalc_x86.elf picocalc_x86.uf2 --family rp2350-riscv --abs-block
/opt/riscv/bin/../lib/gcc/riscv-none-elf/15.2.0/../../../../riscv-none-elf/bin/ld: picocalc_x86.elf section `.bss' will not fit in region `RAM'
/opt/riscv/bin/../lib/gcc/riscv-none-elf/15.2.0/../../../../riscv-none-elf/bin/ld: section .scratch_x VMA [20080000,2008057f] overlaps section .bss VMA [200159d8,2008a533]
/opt/riscv/bin/../lib/gcc/riscv-none-elf/15.2.0/../../../../riscv-none-elf/bin/ld: region `RAM' overflowed by 42292 bytes
Memory region         Used Size  Region Size  %age Used
           FLASH:      374352 B         4 MB      8.93%
           PSRAM:           0 B          0 B
         XIP_RAM:           0 B        16 KB      0.00%
             RAM:      566580 B       512 KB    108.07%
       SCRATCH_X:        3456 B         4 KB     84.38%
       SCRATCH_Y:          32 B         4 KB      0.78%
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
build finished with error(s).

```

--- a/src/rp2_common/pico_standard_link/script_include/default_rodata_excludes.incl
+++ b/src/rp2_common/pico_standard_link/script_include/default_rodata_excludes.incl
@@ -1,1 +1,1 @@
-*(EXCLUDE_FILE(*libgcc.a: *lib*_a-mem*.o *libm.a:) .rodata*)
+*(EXCLUDE_FILE(*libgcc.a: *libm.a:) .rodata*)

--- a/src/rp2_common/pico_standard_link/script_include/default_rodata_excludes.incl
+++ b/src/rp2_common/pico_standard_link/script_include/default_rodata_excludes.incl
@@ -1,1 +1,1 @@
-*(EXCLUDE_FILE(*libgcc.a: *libc.a: *lib*_a-mem*.o *libm.a:) .rodata*)
+*(EXCLUDE_FILE(*libgcc.a: *lib*_a-mem*.o *libm.a:) .rodata*)

--- a/src/rp2_common/pico_standard_link/script_include/default_text_excludes.incl
+++ b/src/rp2_common/pico_standard_link/script_include/default_text_excludes.incl
@@ -1,1 +1,1 @@
-*(EXCLUDE_FILE(*libgcc.a: *libc.a: *lib*_a-mem*.o *libm.a:) .text*)
+*(EXCLUDE_FILE(*libgcc.a: *lib*_a-mem*.o *libm.a:) .text*)

--- a/src/rp2_common/pico_standard_link/script_include/default_text_excludes.incl
+++ b/src/rp2_common/pico_standard_link/script_include/default_text_excludes.incl
@@ -1,1 +1,1 @@
-*(EXCLUDE_FILE(*libgcc.a: *lib*_a-mem*.o *libm.a:) .text*)
+*(EXCLUDE_FILE(*libgcc.a: *libm.a:) .text*)

